src/core/os/amdgpu/amdgpuDevice.cpp

```diff
@@ -1986,6 +2005,25 @@ Result Device::CreateInternalImage(
     return result;
 }
 
+// =====================================================================================================================
+// Returns true if gpu + amdgpu kms driver do support 16 bit floating point display.
+bool Device::HasFP16DisplaySupport(void)
```

Comment:
Coding guidelines:  
"FP16" should instead be "Fp16".  I know its odd, but we handle the UpperCamelCase for acronyms and initialisms the same as we do for normal words.

Also, the "void" in the argument list should be omitted.  Our style guidelines don't include that.

Reply:
Done in force-push just now.

For efficiency, it would be helpful to external contributors if these style guidelines would be publicly available, unless i missed them?